book/07-git-tools/sections/replace.asc

```diff
@@ -89,18 +89,18 @@ $ echo 'get history from blah blah blah' | git commit-tree 9c68fdc^{tree}
 622e88e9cbfbacfb75b5279245b9fb38dfea10cf
 ----
 
-[NOTE]
+[TANDAAN]
 =====
-The `commit-tree` command is one of a set of commands that are commonly referred to as 'plumbing' commands.
-These are commands that are not generally meant to be used directly, but instead are used by *other* Git commands to do smaller jobs.
-On occasions when we're doing weirder things like this, they allow us to do really low-level things but are not meant for daily use.
+Ang `commit-tree` na utos ay isa sa isang set ng mga utos na karaniwang tinutukoy bilang 'plumbing' na mga utos.
+Ang mga utos na ito ay hindi sa pangkalahatang para sa gumagamit nang diretso, ngunit sa halip ay ginagamit ng *ibang* mga utos ng Git na gumawa ng maliliit na mga trabaho.
+Sa mga okasyon kapag kami ay gumawa ng mahiwang mga bagay na tulad nito, pinapahintulotan tayo na gumawa ng mababang antas ngunit hindi nakalaan para sa araw-araw na gamit.
```

Comment:
`araw-araw na gamit` to `pang-araw-araw na paggamit`

Reply:
done change